shiny/render/transformer/_transformer.py

```diff
@@ -220,6 +221,7 @@ def __init__(
         value_fn: ValueFn[IT],
         transform_fn: TransformFn[IT, P, OT],
         params: TransformerParams[P],
+        default_output: Callable[[str], TagChild] | None = None,
```

Comment:
I don't feel super strongly about this, but the name `default_output` makes me think that it's referring to an output, rather than a function that generates a corresponding output UI component. What do you think about calling it `default_output_ui`?

Reply:
Sure, that sounds good